parking_permits/services/talpa.py

```diff
@@ -41,3 +50,68 @@ def snake_to_camel_dict(dictionary):
 def camel_str(snake_str):
     first, *others = snake_str.split("_")
     return "".join([first.lower(), *map(str.title, others)])
+
+
+def create_talpa_order(customer):
+    data = {
+        "customer": {
+            "first_name": customer.first_name,
+            "last_name": customer.last_name,
+            "email": customer.email,
+        },
+        "items": [],
+    }
+    permits = ParkingPermit.objects.filter(
+        customer=customer, status=ParkingPermitStatus.DRAFT
+    )
+    address = (
+        customer.primary_address
+        if permits.first().parking_zone == customer.primary_address.zone
+        else customer.other_address
+    )
+
+    for permit in permits:
+        total_price, monthly_price = permit.get_prices()
+        month_count = permit.month_count
+        open_ended_fields = {
+            "period_unit": "monthly",
+            "period_frequency": "1",
+        }
+        order_line_data = {
+            "quantity": 1
+            if permit.contract_type == ContractType.OPEN_ENDED
+            else month_count,
+            "product_id": str(address.zone.shared_product_id),
+            "product_name": f"{address.zone.name} ({permit.vehicle.registration_number})",
+            "unit": "kk",
+            "meta": [{"key": "permitId", "value": str(permit.id)}],
+        }
+        order_line_data.update(resolve_price_response(total_price, monthly_price))
+        if permit.contract_type == ContractType.OPEN_ENDED:
+            order_line_data.update(open_ended_fields)
+        data["items"].append(order_line_data)
+
+    data.update(
+        {
+            "namespace": settings.NAMESPACE,
+            "user": str(customer.id),
+            "price_net": float(sum([item["price_net"] for item in data["items"]])),
+            "price_vat": float(sum([item["price_vat"] for item in data["items"]])),
+            "price_total": float(sum([item["price_gross"] for item in data["items"]])),
+        }
+    )
+
+    headers = {
+        "api-key": settings.TALPA_API_KEY,
+        "namespace": settings.NAMESPACE,
+        "Content-type": "application/json",
+    }
+    url = f"{settings.TALPA_ORDER_EXPERIENCE_API}"
+    result = requests.post(
+        url=url, data=json.dumps(snake_to_camel_dict(data)), headers=headers
+    )
+    if result.status_code >= 300:
+        raise OrderCreationFailed(
+            "Failed to create talpa order: {}".format(result.text)
+        )
+    return json.loads(result.text)
```

Comment:
Minor, but the response object has a `json` method  available that returns a dict